server/src/main/java/org/elasticsearch/index/mapper/KeywordFieldMapper.java

```diff
@@ -199,6 +210,7 @@ public KeywordFieldMapper build(BuilderContext context) {
 
         private NamedAnalyzer normalizer = null;
         private boolean splitQueriesOnWhitespace;
+        private Integer lengthLimit;
```

Comment:
We could also keep it as primitive int with `Integer.MAX_VALUE` for the case that is represented as null now.